### DISCUSSION TITLE
specifying s3 encryption keys to use

### DISCUSSION THREAD

**Discussion by ddvlamin** [broadinstitute/cromwell#7112] [Q&A]
Is it possible to specify at workflow runtime (for example through the options file) a particular kms key to use for encrypting the data when downloading from or uploading files to S3 (basically when Cromwell localizes the files and uploads them back to final_workflow_outputs_dir)?

If not what would need to change in the code (roughly) to enable this? I was looking at the code [AWSBatchJob.scala](https://github.com/broadinstitute/cromwell/blob/a69d12ec71bd453bf50be563f0666e7fb65c6874/supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchJob.scala), but was then wondering how I could get access to the workflow options in there?

**Comment by henriqueribeiro** [comment]
hi @ddvlamin. are you in Cromwell's slack? me and some other people have been working on AWS backend for Cromwell and we discuss this in Slack.
